tests/settings.py

```diff
@@ -27,6 +28,10 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "debug_toolbar",
+    # We are not actively using template-partials; we just want more nesting
+    # in our template loader configuration, see
+    # https://github.com/django-commons/django-debug-toolbar/issues/2109
+    "template_partials",
```

Comment:
Should we always be testing with template partials? I think with the django-csp tests, we have specific tests that install the package and test against it. That said, this is likely to be a short-time thing. Both django-csp and template-partials are very likely to be merged to core in the 6.x series.

Reply:
It does feel a bit bad, I agree. Another way to test this would have been to add a mock loader with a `loaders` attribute containing the cached loader configuration and testing the panel like that. I thought it easier and more understandable to add django-template-partials as a test dependency than doing the more involved mocking. 

We will have to revisit this when template-partials is merged (hopefully), either by switching to a different package which also wraps the cached loader or by adding the relevant testing code to our project.